pkg/operator/ceph/csi/spec.go

```diff
@@ -498,6 +522,24 @@ func deleteCSIDriverResources(
 	return succeeded
 }
 
+func applyCephClusterNetworkConfig(objectMeta *metav1.ObjectMeta, rookclientset rookclient.Interface) (bool, error) {
+	cephClusters, err := rookclientset.CephV1().CephClusters(objectMeta.Namespace).List(metav1.ListOptions{})
+	if err != nil {
+		logger.Debugf("No CephClusters found in namespace %s.", objectMeta.Namespace)
+		return false, nil
```

Comment:
```suggestion
		return false, errors.New(...)
```

Return an error otherwise nothing will be printed in `startDrivers()`.

Reply:
done